content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -0,0 +1,278 @@
+
+When available, version is added to logs like any other tag.
+
+At the very least they will be used by the APM and [Dogstatsd clients][6] to tag trace data and statsd metrics with `env`, `service`, and `version`. If enabled, the APM tracer will also inject the values of these variables into your logs.
+
+Depending on your CI/CD pipeline and how you run your service (e.g., as a binary or through a Docker container), you may wish to set these environment variables depending on whether your environment is [containerized](#containerized-environment) or [non-containerized](#non-containerized-environment).
+
+{{< img src="tagging/unified_service_tagging/overview.gif" alt="Unified Service Tagging"  >}}
+
+## Requirements
+
+- Unified service tagging requires knowledge of tagging and assigning tags. If you are unsure of how to configure tags, read the [Getting Started with Tagging][7] guide and [Assigning tags][8] documentation before proceeding to configuration.
+
+- Unified service tagging requires setup of the [Datadog Agent][9].
+
+- This guide requires [Autodiscovery][10] for unified service tagging in a containerized environment.
+
+## Recommendations
+
+If you're using [connected Logs and Traces][11], it is recommended to enable automatic logs injection. The APM tracer will then automatically add these fields to your logs, thereby eliminating manual configuration for those fields elsewhere.
+
+## Configuration
+
+### Containerized environment
+
+#### Autodiscovery
+
+In containerized environments, [Autodiscovery][10] tags `env`, `service`, and `version` from standard labels and environment variables. Since the Agent associates data collected with specific containers, the configuration for the standard tags can conveniently reside within the container's metadata.
+
+Additionally, the Autodiscovered tags for `env`, `service`, and `version` are added to all data generated by integration or logs checks. You will not need to add these tags to each check's configuration.
+
+#### Setting variables
+
+While service is generally "constant", `env` and `version` might be determined at build time or afterwards. If known at build time, these parameters could be "baked" into a container's image. Since the environment variables would become part of the image, they'd be available when the container is started. Consider this as an option if injecting the environment variables later on is not possible.
+
+Conversely, it might be more convenient to inject the `DD_*` environment variables at deploy time. If so, you can leverage the relevant API for your containerized infrastructure.
+
+**Note**: It's recommend to set **all three environment variables at the same time and place** if possible. This helps to control consistency across configuration files for all of your applications.
+
+{{< tabs >}}
+{{% tab "Kubernetes" %}}
+
+##### Pod-level metrics
```

Comment:
I would recommend tabs instead of sections.

Also, setting the tags here will also impact logs and traces right?